cli/src/tasks/migrate.ts

```diff
@@ -46,15 +52,19 @@ const plugins = [
   '@capacitor/text-zoom',
   '@capacitor/toast',
 ];
-const coreVersion = '^5.0.0';
-const pluginVersion = '^5.0.0';
+const coreVersion = 'next'; // TODO: Update when Capacitor 5 releases
+const pluginVersion = 'next'; // TODO: Update when Capacitor 5 releases
 const gradleVersion = '7.5';
 
 export async function migrateCommand(config: Config): Promise<void> {
   if (config === null) {
     fatal('Config data missing');
   }
 
```

Comment:
We have a method for checking core version already called `getCoreVersion`.
The error should be more helpful, telling them to use capacitor CLI 4 to migrate if they are in version 2 or 3.
Also, we should probably allow to run the command even if it’s in version 5 already as last year we broke a few things in the migrate and rerunning it after the fix could fix the problems it caused in previous versions